plugins/include/string.inc

```diff
@@ -168,6 +173,23 @@ native format(output[], len, const format[], any:...);
 native formatex(output[], len, const format[], any:...);
 
 /**
+ * Formats and returns a string according to the AMX Mod X format rules
+ * (see documentation).
+ *
+ * @note Example: menu_additem(menu, fmt("My first %s", "item")).
+ * @note This should be used only for simple and unique inline formatting
+ *       like above example.  Do not use this function to store formated
+ *       string into a variable, use other formating functions.
```

Comment:
Also there are other performance implications on this native (namely the return buffer is temporary for each call) but I'd really not want to go that deep on the documentation for this. Our docs are detailed for minute performance penalties, sometimes to their detriment.